src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -29,11 +31,13 @@ using StringInputs = std::variant<std::string, std::vector<std::string>>;
 *
 * @param tokens sequence of resulting tokens
 * @param scores sum of logarithmic probabilities of all tokens in the sequence
+* @param metrics performance metrics with tpot, ttft, etc. of type ov::genai::PerfMetrics
```

Comment:
is it param? typically, doxygen documentation for fields is above those fields, bot outside of the class.